components/electric/lib/electric/ddlx/parse/parser.ex

```diff
@@ -0,0 +1,108 @@
+defmodule Electric.DDLX.Parse.Parser do
+  alias Electric.DDLX.Parse.EnableParser
+  alias Electric.DDLX.Parse.ElectrifyParser
+  alias Electric.DDLX.Parse.DisableParser
+  alias Electric.DDLX.Parse.UnelectrifyParser
+  alias Electric.DDLX.Parse.AssignParser
+  alias Electric.DDLX.Parse.UnassignParser
+  alias Electric.DDLX.Parse.GrantParser
+  alias Electric.DDLX.Parse.RevokeParser
+  alias Electric.DDLX.Parse.SQLiteParser
+  alias Electric.DDLX.Parse.EnableParser
+  alias Electric.DDLX.Parse.Element
+  alias Electric.DDLX.Command
```

Comment:
Such lists are easier to parse (by eye) when they are sorted.

```suggestion
  alias Electric.DDLX.Command
  alias Electric.DDLX.Parse.AssignParser
  alias Electric.DDLX.Parse.DisableParser
  alias Electric.DDLX.Parse.ElectrifyParser
  alias Electric.DDLX.Parse.Element
  alias Electric.DDLX.Parse.EnableParser
  alias Electric.DDLX.Parse.EnableParser
  alias Electric.DDLX.Parse.GrantParser
  alias Electric.DDLX.Parse.RevokeParser
  alias Electric.DDLX.Parse.SQLiteParser
  alias Electric.DDLX.Parse.UnassignParser
  alias Electric.DDLX.Parse.UnelectrifyParser
```